src/OpenAL/OpenToolkit.OpenAL/ALDevice.cs

```diff
@@ -0,0 +1,54 @@
+﻿//
+// ALDevice.cs
+//
+// Copyright (C) 2019 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace OpenToolkit.Audio.OpenAL
+{
+    /// <summary>
+    /// Opaque handle to an OpenAL device.
+    /// </summary>
+    public struct ALDevice : IEquatable<ALDevice>
+    {
+        public static readonly ALDevice Null = new ALDevice(IntPtr.Zero);
```

Comment:
Just a question. Would `NullDevice` sound better here?

Reply:
When you use this it looks like `ALDevice.Null`, I think that gives a pretty good idea of what you are getting. But that's just my opinion

Reply:
I know what you mean. Ah let's keep it Null.